kumascript/macros/NonStandardBadge.ejs

```diff
@@ -41,7 +41,7 @@ var titleAttrValue = mdn.localString({
     "en-US": "This API has not been standardized."
 });
 %>
-<svg class="icon experimental" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"
+<svg class="icon nonStandard" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"
```

Comment:
I believe the reason I did that is that they use the same icon but, perhaps it does make sense to have a separate class should we want to change that in the future. Either way, I would prefer to move away from the casing used and change it to `non-standard` as opposed to `nonStandard`. Thank you @escattone

Reply:
`non-standard` it is!  Thanks @schalkneethling